testsuite/utils/http.py

```diff
@@ -123,7 +123,7 @@ def form(self):
                     keep_blank_values=True,
                     encoding=charset,
                 )
-                self._form = {key: value for key, value in items}
+                self._form = dict(items)
```

Comment:
you can also write `{**items}` so no globals() name lookup is perfromed

Reply:
The return value of `urllib.parse.parse_qsl` is a list (https://docs.python.org/3/library/urllib.parse.html), so trying to unpack it like this `{**items}` would probably leads to `TypeError: 'list' object is not a mapping`.
Based on the practice of using `urllib.parse.parse_qsl` (https://www.programcreek.com/python/example/56006/urllib.parse.parse_qsl), it's better to use `dict(items)`.

Reply:
OK